lib/jobs/base-job.js

```diff
@@ -569,7 +569,7 @@ function baseJobFactory(
      */
     BaseJob.prototype._publishRunRedfishCommand =
         function _publishRunRedfishCommand (uuid, hostData) {
-        return messenger.publish(
+        return messenger.publishInternalEvents(
             Constants.Protocol.Exchanges.Task.Name,
```

Comment:
i have a little doubt about this comment. event.js is used for external event, while the function here is used in job level to communicate among processes inside rackhd. keeping it here is clear in the code structure.